build.gradle

```diff
@@ -168,13 +168,41 @@ def dbTest = tasks.register("dbTest", Test) {
         }
 }
 
+def ccapTest = tasks.register("ccapTest", Test) {
+    Test task ->
+        task.useJUnitPlatform {
+            JUnitPlatformOptions options ->
+                options.includeTags 'ccap'
+        }
+}
+
+def validationTest = tasks.register("validationTest", Test) {
+    Test task ->
+        task.useJUnitPlatform {
+            JUnitPlatformOptions options ->
+                options.includeTags 'validation'
+        }
+}
+
+
+def documentTest = tasks.register("documentTest", Test) {
+    Test task ->
+        task.useJUnitPlatform {
+            JUnitPlatformOptions options ->
+                options.includeTags 'document'
+        }
+}
+
 def testAll = tasks.register("testAll", Test) {
     Test task ->
-        project.logger.lifecycle("Consider using the :test task instead of :testAll!")
+        project.logger.lifecycle("Please use the :test task instead of :testAll!")
         task.dependsOn(unitTest)
         task.dependsOn(journeyTest)
         task.dependsOn(dbTest)
         task.dependsOn(pdfTest)
+        task.dependsOn(ccapTest)
+        task.dependsOn(validationTest)
+        task.dependsOn(documentTest)
```

Comment:
@andrewedstrom Can we remove this now or were you just waiting to merge it first?

Reply:
Doing it now